sdk/src/client/api/block_builder/mod.rs

```diff
@@ -24,15 +24,10 @@ impl Client {
         let issuance = self.get_issuance().await?;
 
         let issuing_time = {
-            #[cfg(feature = "std")]
             let issuing_time = std::time::SystemTime::now()
                 .duration_since(std::time::UNIX_EPOCH)
                 .expect("Time went backwards")
                 .as_nanos() as u64;
-            // TODO no_std way to have a nanosecond timestamp
-            // https://github.com/iotaledger/iota-sdk/issues/647
-            #[cfg(not(feature = "std"))]
-            let issuing_time = 0;
```

Comment:
This is also an issue for wasm? https://github.com/rust-lang/rust/issues/48564

Reply:
used instant instead